torch_xla/csrc/ir_builder.h

```diff
@@ -0,0 +1,143 @@
+#include "tensorflow/compiler/xla/xla_client/debug_macros.h"
+#include "torch/csrc/lazy/core/ir.h"
+#include "torch/csrc/lazy/core/ir_builder.h"
+#include "torch_xla/csrc/ops/as_strided.h"
+#include "torch_xla/csrc/ops/cast.h"
+#include "torch_xla/csrc/ops/diagonal.h"
+#include "torch_xla/csrc/ops/expand.h"
+#include "torch_xla/csrc/ops/generic.h"
+#include "torch_xla/csrc/ops/ops.h"
+
+namespace torch_xla {
+
+struct XLAIrBuilder : IrBuilder {
+  NodePtr MakeDeviceData(
+      const std::shared_ptr<BackendData>& data) const override {
+    return torch::lazy::MakeNode<DeviceData>(data);
```

Comment:
nit: regarding explicit namespaces, should we explicitly define namespaces for `torch::lazy` objects such as `NodePtr` and `Value` like `torch::lazy::MakeNode`?

Reply:
Oh right, yea. I should use `torch::lazy::NodePtr` instead.